src/Form/Field/ObjectChoiceList.php

```diff
@@ -71,34 +72,47 @@ public function getStructuredValues()
     }
 
     /**
-     * {@inheritdoc}
+     * {@inheritDoc}
```

Comment:
remove {@inheritDoc} ?

Reply:
Referering to: Symfony\Component\Form\ChoiceList\ChoiceListInterface methods